test/testenv/util.go

```diff
@@ -43,9 +43,10 @@ const (
 
 func init() {
 	rand.Seed(time.Now().UnixNano())
-	l := zap.LoggerTo(ginkgo.GinkgoWriter)
+	logf.SetLogger(zap.New(zap.WriteTo(ginkgo.GinkgoWriter), zap.UseDevMode(true)).WithName("util"))
+	/*l := zap.LoggerTo(ginkgo.GinkgoWriter)
```

Comment:
nit: remove?

Reply:
yes I would like to remove these comments, need to work with param on this.

Reply:
removed